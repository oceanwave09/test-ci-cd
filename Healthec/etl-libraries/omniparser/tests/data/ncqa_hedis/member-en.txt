95000           1955072220231231YYYYYYYYPOSY         A
95001           1936061120231231YNYYYNNYMP Y         A
95002           1954071220220207NNYYYYYYSN2N         A
95002           2022020820231231NNYYYYYYMCSN         A
95003           2005082220231231YNYYYNYYMMPN         A