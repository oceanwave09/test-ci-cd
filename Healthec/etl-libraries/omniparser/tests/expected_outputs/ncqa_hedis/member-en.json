[
	{
		"chemdep_benefit_inpatient": "Y",
		"chemdep_benefit_intensive_outpatient": "Y",
		"chemdep_benefit_outpatient_ed": "Y",
		"dental_benefit": "Y",
		"disenrollment_date": "20231231",
		"drug_benefit": "Y",
		"health_plan_employee_flag": "Y",
		"indicator": "A",
		"member_id": "95000",
		"mental_health_benefit_inpatient": "Y",
		"mental_health_benefit_intensive_outpatient": "Y",
		"mental_health_benefit_outpatient_ed": "Y",
		"payer": "POS",
		"row_id": "81dcf966-8c9d-4365-b44a-3359d44c3db8",
		"start_date": "19550722"
	},
	{
		"chemdep_benefit_inpatient": "N",
		"chemdep_benefit_intensive_outpatient": "N",
		"chemdep_benefit_outpatient_ed": "Y",
		"dental_benefit": "Y",
		"disenrollment_date": "20231231",
		"drug_benefit": "N",
		"health_plan_employee_flag": "Y",
		"indicator": "A",
		"member_id": "95001",
		"mental_health_benefit_inpatient": "Y",
		"mental_health_benefit_intensive_outpatient": "Y",
		"mental_health_benefit_outpatient_ed": "Y",
		"payer": "MP",
		"row_id": "05c4a85f-ba98-4468-84c5-e4e1f13a34eb",
		"start_date": "19360611"
	},
	{
		"chemdep_benefit_inpatient": "Y",
		"chemdep_benefit_intensive_outpatient": "Y",
		"chemdep_benefit_outpatient_ed": "Y",
		"dental_benefit": "N",
		"disenrollment_date": "20220207",
		"drug_benefit": "N",
		"health_plan_employee_flag": "N",
		"indicator": "A",
		"member_id": "95002",
		"mental_health_benefit_inpatient": "Y",
		"mental_health_benefit_intensive_outpatient": "Y",
		"mental_health_benefit_outpatient_ed": "Y",
		"payer": "SN2",
		"row_id": "ce5b7942-8fff-45bb-9a8c-62fe5ba1a953",
		"start_date": "19540712"
	},
	{
		"chemdep_benefit_inpatient": "Y",
		"chemdep_benefit_intensive_outpatient": "Y",
		"chemdep_benefit_outpatient_ed": "Y",
		"dental_benefit": "N",
		"disenrollment_date": "20231231",
		"drug_benefit": "N",
		"health_plan_employee_flag": "N",
		"indicator": "A",
		"member_id": "95002",
		"mental_health_benefit_inpatient": "Y",
		"mental_health_benefit_intensive_outpatient": "Y",
		"mental_health_benefit_outpatient_ed": "Y",
		"payer": "MCS",
		"row_id": "d56fb3ba-10ab-4d0b-a796-000e98de790d",
		"start_date": "20220208"
	},
	{
		"chemdep_benefit_inpatient": "N",
		"chemdep_benefit_intensive_outpatient": "Y",
		"chemdep_benefit_outpatient_ed": "Y",
		"dental_benefit": "Y",
		"disenrollment_date": "20231231",
		"drug_benefit": "N",
		"health_plan_employee_flag": "N",
		"indicator": "A",
		"member_id": "95003",
		"mental_health_benefit_inpatient": "Y",
		"mental_health_benefit_intensive_outpatient": "Y",
		"mental_health_benefit_outpatient_ed": "Y",
		"payer": "MMP",
		"row_id": "ce672c04-f7d4-4c25-93a7-1069e99687fc",
		"start_date": "20050822"
	}
]
